docs/hub/timm.md

```diff
@@ -0,0 +1,162 @@
+# Using Timm at Hugging Face
+
+`timm`, also known as [pytorch-image-models](https://github.com/rwightman/pytorch-image-models), is an open-source collection of state-of-the-art PyTorch image models, pretrained weights, and utility scripts for training, inference, and validation.
+
+## Exploring Timm in the Hub
+
+You can find a number of `timm` models by filtering at the left of the [models page](https://huggingface.co/models?library=timm&sort=downloads).
+
+All models on the Hub come up with useful feature:
+1. An automatically generated model card, which model authors can complete with [information about their model](./models-cards).
+2. Metadata tags that help for discoverability, allowing users to find the model when searching for `timm` models.
+3. An [interactive widget](./models-widgets) you can use to play out with the model directly in the browser
+4. An [Inference API](./models-inference) that allows users to make inference requests
+
+## Using existing models
+
+Any `timm` model from the Hugging Face Hub can be loaded with a single line of code as long as you have `timm` installed! Once you've selected a model from the Hub, pass the model's ID prefixed with `hf_hub:` to `timm`'s `create_model` method to download and instantiate the model. 
+
+```py
+import timm
+
+# Loading https://huggingface.co/sgugger/resnet50d
+model = timm.create_model("hf_hub:sgugger/resnet50d", pretrained=True)
+```
+
+If you want to see how to load a specific model, you can click **Use in timm** and you will be given a working snippet that you can load it! 
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-timm_snippet1.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-timm_snippet1-dark.png"/>
+</div>
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-timm_snippet2.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-timm_snippet2-dark.png"/>
+</div>
+
+### Inference
+
+The snippet below shows how you can perform inference on a `timm` model loaded from the Hub:
+
+```py
```

Comment:
Is this snippet too long? I wanted a full example - this is basically exactly what the `timm` [image classification pipeline](https://github.com/huggingface/api-inference-community/blob/main/docker_images/timm/app/pipelines/image_classification.py) does.

Reply:
also should double check that the pipeline is up to date...did some different things here that aren't 100% aligned with whats going on there

Reply:
Nah I think the snippet's the right length. Especially since this stuff isn't really documented anywhere else I think it's good to have it here right now. Later, when we port over the time docs to hf.co/doc we can look at shortening it here and just linking people to hf.co/docs/timm